linux57-tkg/customization.cfg

```diff
@@ -77,9 +79,6 @@ _ftracedisable="false"
 # Set to "true" to disable NUMA, lowering overhead, but breaking CUDA/NvEnc on Nvidia equipped systems - Kernel default is "false"
 _numadisable="false"
 
-# Set to "true" to enable misc additions - May contain temporary fixes pending upstream or changes that can break on non-Arch - Kernel default is "true"
-_misc_adds="true"
```

Comment:
This is here to stay, even if I end up removing the specific patch it was initially introduced for.

Reply:
Yea I wanted to discuss that with you. As I understand now you didn't put it only for Ubuntu to be able to compile, then okay! 

Are you the one who reverts commits or me ?

Reply:
Yeah it was to have a place to add temporary patches without using -base for it. The decoupling of "that patch" with -base was following that logic of temporary fixing the issue we had with it.

I have no power on the PR's code until it's effectively merged afaik.

Reply:
I actually gave you push permissions hehe

Reply:
I have a small concern about the misc patch, I would like people from ubuntu for example to get the amdgpu patch but not have the change that gives the leaky tty :/